### PR TITLE
Tidy up markdown documentation

### LINKED ISSUE #97: Tidy up markdown documentation
`README.md`, `CONTRIBUTING.md`, `INSTALL.md`, and `DOCKER.md` could all use a careful read-through to polish up language and clarify missing or ambiguous bits of information.

### PR DESCRIPTION
**Pull request by mcevoypeter**
## Description
Resolves #97.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -101,15 +112,6 @@ Contributions:
...
-
-### Announce the update
-
-Post an announcement to urbit-dev.  The tag annotation, basically, is fine here
--- I usually add the release binary URLs.  Check the urbit-dev archives for examples 
-of these announcements.
-
-Post the same announcement to the group feed of Urbit Community.
+TODO:
+- Do we *need* to upload tarballs of the released binaries to the GitHub
```
[the remark this answers is not in the document]
Comment (mcevoypeter) [reply MAINTAINERS.md:116]: We'll keep uploading the tarballs. It's not hard to generate them, and I may even be able to automate their attachment to the release notes somehow.

**Comment by mcevoypeter** [comment]
@zalberico I'm going to make a few additions to `MAINTAINERS.md`, and then I'll merge it.

**Comment by mcevoypeter** [comment]
@zalberico can you take one last look? I added more explicit instructions about releasing to the end of `MAINTAINERS.md`.

**Comment by zalberico** [comment]
Maybe we should add a small section about docker to `MAINTAINERS.md` since that seems like something a lot of people care about?

Is there anything else from this week people asked a lot about that we're not covering here? Docker is the main thing I can think of, this doc looks good to me otherwise. In any case, I think it's okay to merge it and then submit follow on PRs (since it's already better than what's there as is).

File: MAINTAINERS.md [hunk 2]
```diff
@@ -1,115 +1,129 @@
...
+
+Each time a release is cut (i.e. `develop` is merged into `release` to kick off
+a release), the version number should be bumped on `develop` in anticipation of
+the next release.
+
+## Deploy Endpoints
+
+Binaries are deployed to the following endpoints, where `{VN}` is the version
+number in VERSION, `{CS}` is the shortened commit SHA of the commit the binary
+is built off, and `{P}` is one of `linux-aarch64`, `linux-x86_64`,
```
Comment (jalehman) [inline MAINTAINERS.md:65]:
```suggestion
is built from, and `{P}` is one of `linux-aarch64`, `linux-x86_64`,
```

File: MAINTAINERS.md [hunk 3]
```diff
@@ -1,115 +1,129 @@
...
+PR merges--we build and deploy a new version of the binary available for
+consumption by anyone subscribed to that train (via `<pier>/.bin/pace`).
+
+For `edge` and `soon`, each binary is given a version of the form
+`{version number}-{shortened commit SHA}`, where `{version number}` is the
+version number listed in the [version file in the root of this repo](./VERSION)
+and `{shortened commit SHA}` is the shortened commit SHA of the commit the
+binary was built from. This allows subscribers to `edge` and `soon` to
+continually pull down new binaries via the `next` subcommand even when the
+version number in the [version file](./VERSION) remains the same.
```
Comment (jalehman) [inline MAINTAINERS.md:51]: This is great.

**Review by jalehman — APPROVED** [review]

File: MAINTAINERS.md (hunk 2 above)
Comment (mcevoypeter) [reply MAINTAINERS.md]: Good catch. Thanks.

**Comment by mcevoypeter** [comment]
> Maybe we should add a small section about docker to `MAINTAINERS.md` since that seems like something a lot of people care about?

Tracking issue: #118. I'll address once we resolve #112.

**Pull request merged**